### PR TITLE
feat: Changed Stepper, Prompt from Vue 2 to Vue 3

### PR DESCRIPTION
**Pull request by Palaksharma23**
I have changed Stepper, Prompt component from Vue 2 to Vue 3 

@HarshCasper may you please once review this.

### CODE REVIEW

**Review by HarshCasper — APPROVED** [review]

**Review by YashKandalkar — COMMENTED** [review]
Hi - @Palaksharma23 don't we have to provide an `emits` array with all the events that are emitted by a child component in Vue 3?

File: flint.ui/src/components/Stepper/StepperPoint.vue [hunk 1]
```diff
@@ -10,10 +10,13 @@
 
 <script>
 export default {
-  methods: {
-    finalPool() {
+  setup() {
+  function finalPool() {
```
Comment (YashKandalkar) [inline flint.ui/src/components/Stepper/StepperPoint.vue:14]: The indentation doesn't look right here, can you fix this?

**Review by YashKandalkar — CHANGES_REQUESTED** [review]

**Review by YashKandalkar — APPROVED** [review]
LGTM :rocket: 
Thanks @Palaksharma23 !

**Comment by YashKandalkar** [comment]
@Palaksharma23 the build is failing - looks like there are some linting issues, can you have a look?

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**